nativeruntime/src/main/java/org/robolectric/nativeruntime/NativeRuntimeLoader.java

```diff
@@ -50,8 +50,13 @@ private static boolean isSupported() {
   }
 
   private static String nativeLibraryPath() {
+    String os = osName();
+    String arch = arch();
+    if (os.equals("mac")) {
+      arch = "universal"; // use the universal library
```

Comment:
Does it need to compare the performance between old arm64 only build and current universal build? Anyway, `universal` can be a game changer to help us to integrate macOS arm64 building at CI.

Reply:
I would assume the performance is identical, as it's the same native code.